apps/wallet/src/features/Explorer/Header.tsx

```diff
@@ -0,0 +1,100 @@
+import { StyleSheet, Text } from 'react-native';
+import { Hoverable, View } from '@walless/gui';
+import { Eye, EyeOff, Settings } from '@walless/icons';
+import { getValuationDisplay } from 'features/Widget/BuiltInNetwork/WalletCard/Balance';
+import { appState } from 'state/app';
+import { setPrivacy } from 'state/runtime/config';
+import { useTokens } from 'utils/hooks';
+import { navigate } from 'utils/navigation';
+import { useSnapshot } from 'valtio';
+
+const Header = () => {
+	const { config } = useSnapshot(appState);
+	const { valuation } = useTokens();
+
+	const handleNavigateToSettings = () => {
+		navigate('Dashboard', {
+			screen: 'Explore',
+			params: {
+				screen: 'Widget',
+				params: {
+					screen: 'Setting',
+				},
+			},
+		});
+	};
```

Comment:
because the navigation tree on mobile is different from the extension, please double-check and confirm that this navigate working fine on both platforms

Reply:
it completely works well